### ISSUE TITLE
Multiple VRChat Crashes

### ISSUE THREAD

**Issue by pinhead-tf2** [Xiexe/XSOverlay-Issue-Tracker#191]
**Describe the bug in detail:**
Overlays frequently crash in VRChat whenever I try to interact with them

**Steps to reproduce the behavior:**
Play VRChat and try and use some other windows

**Output Log:**
I have given 3 logs to use. 

[output_log_2021-10-29__21.33.31.txt](https://github.com/Xiexe/XSOverlay-Issue-Tracker/files/7445716/output_log_2021-10-29__21.33.31.txt)
[output_log_2021-10-29__21.40.18.txt](https://github.com/Xiexe/XSOverlay-Issue-Tracker/files/7445717/output_log_2021-10-29__21.40.18.txt)
[output_log_2021-10-29__21.32.00.txt](https://github.com/Xiexe/XSOverlay-Issue-Tracker/files/7445718/output_log_2021-10-29__21.32.00.txt)

**Comment by pinhead-tf2** [comment]
Now that I’m not tired from staying up too late, it appears that it crashes on any game, with the same errors. I’ve also noticed that the window it’s attempting to capture will be frozen, and interacting with the window or window select will cause the crash.

**Comment by XenoxUSser** [comment]
> Now that I’m not tired from staying up too late, it appears that it crashes on any game, with the same errors. I’ve also noticed that the window it’s attempting to capture will be frozen, and interacting with the window or window select will cause the crash.

you probably using a quest with airlink can that be ?

**Comment by pinhead-tf2** [comment]
Nope, Oculus Rift S running latest software updates, same goes for SteamVR, Windows, and graphics drivers.
